js/interface.js

```diff
@@ -231,7 +230,6 @@ var DynamicLists = (function() {
           var value = $(this).val();
           var type = $(this).data('field');
           var $selector = $(this).parents('.filter-panel');
-          var id = $(this).attr('filter-item-id');
```

Comment:
@tonytlwu i saw your comment but, i created this revert using `git revert command` and use doc which you give's me.
i reverted to the commit https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/454/commits/1603aa15b85d9df695a59954987c6bfb9dde1ca3, so i hope this is the correct revert